config/initializers/datadog_apm.rb

```diff
@@ -4,9 +4,17 @@
 if ENV.fetch('DATADOG_ENABLE_APM', 'false') != 'false'
   require 'ddtrace'
 
+  Rails.logger.info("Enabling Datadog APM tracer")
+
   Rails.configuration.datadog_trace = {
     auto_instrument: true,
     default_service: 'samson',
-    trace_agent_hostname: ENV.fetch('STATSD_HOST', 'localhost')
+    default_controller_service: 'samson-rails-controller',
+    default_cache_service: 'samson-cache',
+    default_database_service: 'samson-mysql',
+    trace_agent_hostname: ENV.fetch('STATSD_HOST', 'localhost'),
```

Comment:
seems a bit odd that there is no `trace_agent_port`

Reply:
There is, I'm just not overriding it. I could add that line for completeness.